x-pack/solutions/search/plugins/search_indices/public/components/create_index/create_index_page.tsx

```diff
@@ -18,6 +18,7 @@ import { LoadIndicesStatusError } from '../shared/load_indices_status_error';
 import { CreateIndex } from './create_index';
 import { usePageChrome } from '../../hooks/use_page_chrome';
 import { IndexManagementBreadcrumbs } from '../shared/breadcrumbs';
+import { css } from '@emotion/react';
```

Comment:
is this being used? or is it needed for some other reason?

Reply:
Whoops, leftovers, removed!